### PR TITLE
Update mcc-isp-faq.yml

### PR DESCRIPTION
**Pull request by andyrivMSFT**
Adding clarification around OS requirements.

## Description

Adding clarification around OS requirements.

## Why

Documentation needs to be clear for customers.

## Changes

Added Ubuntu OS version information

### CODE REVIEW

File: windows/deployment/do/mcc-isp-faq.yml [hunk 1]
```diff
@@ -33,12 +33,18 @@ sections:
       - question: What are the prerequisites and hardware requirements?
         answer: | 
          - Azure subscription
```
Comment (JohanFreelancer9) [inline windows/deployment/do/mcc-isp-faq.yml:35]:
```suggestion
         - Azure subscription.
```

File: windows/deployment/do/mcc-isp-faq.yml [hunk 2]
```diff
@@ -33,12 +33,18 @@ sections:
       - question: What are the prerequisites and hardware requirements?
         answer: | 
          - Azure subscription  
-         - Hardware to host Microsoft Connected Cache:   
+         - Hardware to host Microsoft Connected Cache
```
Comment (JohanFreelancer9) [inline windows/deployment/do/mcc-isp-faq.yml:36]:
```suggestion
         - Hardware to host Microsoft Connected Cache.
```

File: windows/deployment/do/mcc-isp-faq.yml [hunk 3]
```diff
@@ -33,12 +33,18 @@ sections:
...
+         - Ubuntu 20.04 LTS on a physical server or VM of your choice. 
+         
+         > [!NOTE]
+         > The Microsoft Connected Cache is deployed and managed using Azure IoT Edge and Ubuntu 20.04 is an [Azure IoT Edge Tier 1 operating system](https://learn.microsoft.com/en-us/azure/iot-edge/support?view=iotedge-2020-11#tier-1). Additionally, the Microsoft Connected Cache module is optimized for Ubuntu 20.04 LTS.
```
Comment (JohanFreelancer9) [inline windows/deployment/do/mcc-isp-faq.yml:40]:
```suggestion
         > The Microsoft Connected Cache is deployed and managed using Azure IoT Edge and Ubuntu 20.04 is an [Azure IoT Edge Tier 1 operating system](https://learn.microsoft.com/azure/iot-edge/support?view=iotedge-2020-11#tier-1). Additionally, the Microsoft Connected Cache module is optimized for Ubuntu 20.04 LTS. 
```

File: windows/deployment/do/mcc-isp-faq.yml [hunk 4]
```diff
@@ -33,12 +33,18 @@ sections:
...
+         > [!NOTE]
+         > The Microsoft Connected Cache is deployed and managed using Azure IoT Edge and Ubuntu 20.04 is an [Azure IoT Edge Tier 1 operating system](https://learn.microsoft.com/en-us/azure/iot-edge/support?view=iotedge-2020-11#tier-1). Additionally, the Microsoft Connected Cache module is optimized for Ubuntu 20.04 LTS. 
+         
+         The following are recommended hardware configurations:   
           
           <!--Using include file, mcc-prerequisites.md, for shared content on DO monitoring-->
           [!INCLUDE [Microsoft Connected Cache Prerequisites](includes/mcc-prerequisites.md)]
 
-         We have one customer who is able to achieve 40-Gbps egress rate using the following hardware specification:  
+         We have one customer who is able to achieve mid-30s Gbps egress rate using the following hardware specification:
```
Comment (JohanFreelancer9) [inline windows/deployment/do/mcc-isp-faq.yml:47]:
```suggestion
         We have one customer who is able to achieve a mid-30s Gbps egress rate using the following hardware specification:  
```

**Review by JohanFreelancer9 — CHANGES_REQUESTED** [review]
Copy review - @andyrivMSFT, kindly commit the suggested changes. Many thanks.

File: windows/deployment/do/mcc-isp-faq.yml (hunk 3 above)
Comment (JohanFreelancer9) [inline windows/deployment/do/mcc-isp-faq.yml:40]:
```suggestion
         > The Microsoft Connected Cache is deployed and managed using Azure IoT Edge and Ubuntu 20.04 is an [Azure IoT Edge Tier 1 operating system](/azure/iot-edge/support?view=iotedge-2020-11#tier-1). Additionally, the Microsoft Connected Cache module is optimized for Ubuntu 20.04 LTS. 
```

**Review by JohanFreelancer9 — CHANGES_REQUESTED** [review]
@andyrivMSFT, one more, please, to change the link to a relative link. Many thanks.

**Review by aczechowski — APPROVED** [review]

**Pull request merged**